superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -1252,6 +1442,23 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
                 />
                 {renderPreferredSelector()}
                 {renderAvailableSelector()}
+                <Upload
+                  name="databaseFile"
+                  id="databaseFile"
+                  data-test="database-file-input"
+                  accept=".yaml,.json,.yml,.zip"
+                  customRequest={() => {}}
+                  onChange={onDbImport}
+                  onRemove={removeFile}
+                >
+                  <Button
+                    data-test="import-database-btn"
+                    buttonStyle="link"
+                    css={buttonLinkStyles}
```

Comment:
Suggestion from Jess- can we make this 14px?

Reply:
Can do! Adjusted in [`this commit`](https://github.com/apache/superset/pull/19314/commits/a50e3dd5c8203438eadb1ebc83929fb070b54228).